bucket/tortoisesvn.json

```diff
@@ -78,9 +78,9 @@
         ]
     ],
     "post_install": [
-        "$escdir = $dir -replace \"\\\\\",\"\\\\\"",
-        "(gc \"$dir\\tortoisesvn-install.reg.templ\") | % {$_ -replace \"%TSVNHOME%\",\"$escdir\"} | out-file -enc oem \"$dir\\tortoisesvn-install.reg\"",
-        "(gc \"$dir\\tortoisesvn-install-tools.reg.templ\") | % {$_ -replace \"%TSVNHOME%\",\"$escdir\"} | out-file -enc oem \"$dir\\tortoisesvn-install-tools.reg\"",
+        "$escdir = $dir -Replace \"\\\\\",\"\\\\\"",
```

Comment:
Operators are not defined with uppercase ..

https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comparison_operators?view=powershell-6

Reply:
Double quotes are not needed